app/templates/courses.hbs

```diff
@@ -33,8 +33,8 @@
         data-test-year-filter
       >
         {{#each (sort-by "title:desc" @model.years) as |year|}}
-          <option value={{year.year}} selected={{is-equal year this.selectedYear}}>
-            {{year.academicYearTitle}}
+          <option value={{year.id}} selected={{is-equal year this.selectedYear}}>
```

Comment:
This sets a variable on the controller called `yearTitle`. I think using the ID is better, but that variable name should change.

Reply:
:heavy_check_mark: